<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="C02C61S0MD6T" name="retryanalyzer" tests="3" failures="3" timestamp="10 Feb 2021 07:56:55 GMT" time="0.005" errors="0">
  <testcase name="testRetry" time="0.0" classname="com.TestNGTutorials.TestNGRetryAnalyzer.RetryTest">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
	at com.TestNGTutorials.TestNGRetryAnalyzer.RetryTest.testRetry(RetryTest.java:9)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- testRetry -->
  <testcase name="testRetry" time="0.001" classname="com.TestNGTutorials.TestNGRetryAnalyzer.RetryTest">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
	at com.TestNGTutorials.TestNGRetryAnalyzer.RetryTest.testRetry(RetryTest.java:9)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- testRetry -->
  <testcase name="testRetry" time="0.0" classname="com.TestNGTutorials.TestNGRetryAnalyzer.RetryTest">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
	at com.TestNGTutorials.TestNGRetryAnalyzer.RetryTest.testRetry(RetryTest.java:9)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- testRetry -->
</testsuite> <!-- retryanalyzer -->
